iDiff/utils/image_prep_utils.go

```diff
@@ -157,11 +133,39 @@ func (p CloudPrepper) ImageToFS() (string, error) {
 	return getImageFromTar(tarPath)
 }
 
+func (p CloudPrepper) getConfig() (ConfigSchema, error) {
+	ref, err := docker.ParseReference("//" + p.Source)
```

Comment:
Looks like in [their code](https://github.com/containers/image/blob/21bb303613ccad2b16732927c95f9493cdf8742b/docker/docker_transport.go#L50) they require it....but then they immediately strip it away? Not at all clear why they do that, oh well